<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TubeStatusDisplay</class>
 <widget class="QGroupBox" name="TubeStatusDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>120</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>0</height>
      </size>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="signalLamp1">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../Image/image.qrc">
         <normaloff>:/red.png</normaloff>:/red.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="signalLamp2">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../Image/image.qrc">
         <normaloff>:/red.png</normaloff>:/red.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="signalLamp3">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../Image/image.qrc">
         <normaloff>:/red.png</normaloff>:/red.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="signalLamp4">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../Image/image.qrc">
         <normaloff>:/red.png</normaloff>:/red.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="signalLamp5">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../Image/image.qrc">
         <normaloff>:/red.png</normaloff>:/red.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Image/image.qrc"/>
 </resources>
 <connections/>
</ui>
